ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3386,6 +3386,19 @@ private final boolean checkChildren (boolean isCommit)
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_57(get_uid());
                 } else {
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_58(get_uid());
+
```

Comment:
Sorry for confusing. I might have read his slide wrong. I've missed the point that he suggests to use tracef and others so that isTraceEnabled wouldn't be needed. I thought jboss logging handles that checking in the background.

Anyway, I don't remember us using isWarnEnabled before.